hal/src/electron/cellular_hal.cpp

```diff
@@ -7,6 +8,36 @@
 
 static CellularCredentials cellularCredentials;
 
+static HAL_NET_Callbacks netCallbacks = { 0 };
+
```

Comment:
disregard the above question. While it's okay to pass more arguments than necessary to a function, it's certainly bad idea to pass fewer arguments, which is the case for dynalib functions without `void* reserved`.